src/App.js

```diff
@@ -14,56 +16,78 @@ class App extends Component {
 
     this.state = {
       web3: null,
-      keyWord: ''
+      keywords: []
     }
-
-    this.getKeyWord = this.getKeyWord.bind(this);
   }
 
-  componentDidMount() {     
+  componentDidMount() {
     getWeb3
-    .then(results => {
-      this.setState({
-        web3: results.web3
+      .then(results => {
+        this.setState({
+          web3: results.web3,
+          statusSeeker: statusSeekerContract.withProvider(results.web3.currentProvider)
+        })
+      })
+      .catch((err) => {
+        console.error(err)
       })
-    })
-    .catch(() => {
-      console.log('Error finding web3.')
-    })
   }
 
-  getKeyWord(e) {
-    e.preventDefault();
+  getKeyWord = async (e) => {
```

Comment:
I know that in the final product, we will not just be passing random `id`s into this function and will be pulling them from QRs, so will have little chance of getting repeat words. However, people might mistakenly scan the same QR twice and we need to make sure we handle that sort of thing gracefully. Currently, when you get a keyword that is the same as one you already have, my console logs this error:
```
warning.js:33 Warning: flattenChildren(...): Encountered two children with the same key, `art`. Child keys must be unique; when two children share a key, only the first child will be used.
    in div (at KeywordOrganizer.js:10)
    in KeywordOrganizer (created by DragDropContext(KeywordOrganizer))
    in DragDropContext(KeywordOrganizer) (at App.js:73)
    in div (at App.js:71)
    in div (at App.js:66)
    in div (at App.js:99)
    in div (at App.js:98)
    in main (at App.js:97)
    in div (at App.js:92)
    in App (at index.js:6)
```

Any chance we could pop up a little alert/modal to tell the user they already have that word, so move on and keep exploring?